Src/Newtonsoft.Json/Utilities/DateTimeUtils.cs

```diff
@@ -73,6 +73,21 @@ public static XmlDateTimeSerializationMode ToSerializationMode(DateTimeKind kind
                     throw MiscellaneousUtils.CreateArgumentOutOfRangeException("kind", kind, "Unexpected DateTimeKind value.");
             }
         }
+#else
+        public static string ToDateTimeFormat(DateTimeKind kind)
+        {
+            switch (kind)
+            {
+                case DateTimeKind.Local:
+                    return "yyyy-MM-ddTHH:mm:sszzzzzzz";
```

Comment:
Why so many `z`'s? https://msdn.microsoft.com/en-us/library/8kb3ddd4 says timezone specifier allows up to 3 `z`'s, having more doesn't seem to make any difference.